site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -32,11 +32,21 @@ public function showQueue() {
             );
         }
 
+        $queries = array();
+        parse_str($_SERVER['QUERY_STRING'], $queries);
+
+        $full_history = false;
+        if (array_key_exists('full_history', $queries) && $queries['full_history'] === "true") {
+            $full_history = true;
+        }
+
+        $this->core->getOutput()->addInternalCss('officeHoursQueue.css');
+
         return Response::WebOnlyResponse(
             new WebResponse(
                 'OfficeHoursQueue',                      //Goes to this file OfficeHoursQueueView.php
                 'showTheQueue',                          //Runs this functin showTheQueue()
```

Comment:
```suggestion
                'showTheQueue',                          //Runs this function showTheQueue()
```

Reply:
Actually I feel the comments are not necessary here as the meaning of `new WebResponse(somearg)` is consistent throughout Submitty. There is nothing special about the usage here.